docs/guide/siteaccess.md

```diff
@@ -66,66 +87,25 @@ ezpublish:
 
 !!! note
 
-    The new SiteAccess should be recognized by the system, however, an anonymous user will not have read access to a new SiteAccess until it is [explicitly given via the Admin > Roles panel](repository.md#use-cases). Without read access the anonymous user will simply be directed to the default login page.
-
-### The default scope
-
-When no particular context is required, it is fine to use the \`default\` scope instead of specifying a siteaccess.
-
-## Configuration
-
-### Basics
-
-Important
-
-Configuration is tightly related to the service container.
-To fully understand the following content, you need to be familiar with [Symfony's service container](Service-Container_31432100.html) and [its configuration](http://symfony.com/doc/current/book/service_container.html#service-parameters).
-
-Basic configuration handling in eZ Platform is similar to what is commonly possible with Symfony. Regarding this, you can define key/value pairs in [your configuration files](http://symfony.com/doc/current/book/service_container.html#importing-other-container-configuration-resources), under the main **parameters** key (like in **[parameters.yml](https://github.com/ezsystems/ezpublish5/blob/master/app/config/parameters.yml.dist)**).
+    A new SiteAccess is recognized by the system, but an anonymous user will not have read access to it until it is [explicitly given via the Admin > Roles panel](repository.md#use-cases). Without read access the anonymous user will simply be directed to the default login page.
```

Comment:
Anonymous User is provided built-in our software. To me, it's a domain name to first UC.